java-client/src/main/java/org/opensearch/client/opensearch/_types/OpensearchException.java

```diff
@@ -40,12 +40,12 @@
  * additional details that depend on the error type and the API endpoint that
  * was called.
  */
-public class ElasticsearchException extends RuntimeException {
+public class OpensearchException extends RuntimeException {
```

Comment:
elasticsearch - opensearch
Elasticsearch - OpenSearch
es - opensearch
ES - OpenSearch

Reply:
Sure, will update it as part of next PR